lib/sift/client.rb

```diff
@@ -219,6 +219,7 @@ def track(event, properties = {}, opts = {})
       return_route_info = opts[:return_route_info]
       force_workflow_run = opts[:force_workflow_run]
       abuse_types = opts[:abuse_types]
+      include_score_percentile = opts[:include_score_percentile]
```

Comment:
lets name it `include_score_percentiles`

Reply:
Ok Noted